### PR TITLE
Product cards done

### PR DESCRIPTION
**Pull request by JeniaGaidukov**
- [DEMO LINK](https://JeniaGaidukov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://JeniaGaidukov.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,63 @@
...
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
```
Comment (marina-tilniak) [inline src/index.html:3]:
```suggestion
```

Comment (marina-tilniak) [reply src/index.html:3]: there  shouldn't be empty line between parent and child tags

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,63 @@
...
+        <img
+          src="images/imac.jpeg"
+          alt="Imac PC"
+          class="product-card__logo-img"
+        >
+      </div>
+      <div class="product-card__content">
+        <div class="product-card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
```
Comment (marina-tilniak) [inline src/index.html:29]: use semantic tag

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,63 @@
...
+        <div class="product-card__code">
+          Код товара: 195434
+        </div>
```
Comment (marina-tilniak) [inline src/index.html:32]: same here, you can use `<p>`

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,147 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: #060b35;
+
+  --small-fonts: #616070;
```
Comment (marina-tilniak) [inline src/style.css:8]: when you call variable like that you expect it to set font-size, not a color

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-of-type(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-of-type(-n+5) {
+  background-image: url(images/star-active.svg);
+}
```
Comment (marina-tilniak) [inline src/style.css:89]:
```suggestion
.stars--1 .stars__star:nth-child(1),
.stars--2 .stars__star:nth-of-type(-n+2),
.stars--3 .stars__star:nth-of-type(-n+3),
.stars--4 .stars__star:nth-of-type(-n+4),
.stars--5 .stars__star:nth-of-type(-n+5) {
  background-image: url(images/star-active.svg);
}
```

**Review by marina-tilniak — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 5 above)
Comment (JeniaGaidukov) [reply src/style.css]: Виправив, перевірте будь-ласка, дякую)

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,62 @@
...
+        <img
+          src="images/imac.jpeg"
+          alt="Imac PC"
+          class="product-card__logo-img"
+        >
+      </div>
+      <div class="product-card__content">
+        <p class="product-card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
```
Comment (marina-tilniak) [inline src/index.html:28]: це більше схоже на заголовок

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,62 @@
...
+          <a href="#"
+            class="product-card__btn" data-qa="hover">
+            Купить
+          </a>
+        </div>
+      </div>
+    </article>
+  </body>
+
 </html>
```
Comment (marina-tilniak) [inline src/index.html:62]:
```suggestion
  </body>
</html>
```

Comment (marina-tilniak) [reply src/index.html:62]: There shouldn't be empty line between parent and child elements

**Review by marina-tilniak — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 7 above)
Comment (JeniaGaidukov) [reply src/index.html]: Виправив.

**Review by Bohdan-Maksymiuk — APPROVED** [review]

**Pull request closed without merging**